### DISCUSSION TITLE
🔗 All options needed to customize Pagination and Navigation in Loop carousel and Nested carousel widgets. (Duplicate)

### DISCUSSION THREAD

**Discussion by Michaelgimii** [elementor/elementor#22659] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi elementor team.

As you have plans to improve control over Pagination and Navigation in carousels, I wanted to point out the options that are essential for these two areas and hope you consider them. Please note that these allow us to have unlimited customization on the Pagination and Navigation in the Carousel widgets. These items are already in the Jet Woobuilder plugin and in fact they are not a special feature.

### Describe the solution you'd like

**Since it is better for me to add an image for each option, it is better to add the images as links. This may be inconvenient for you and I apologize in advance for that.**

### Pagination

1. Ability to control color in active state : https://prnt.sc/rojcIMBwm4qh
2. gap control to set the spacing between pagination : https://prnt.sc/DQsRJ-EJldD3
3. width and height : https://prnt.sc/WEQM0N0VmBTu
4. border and border randios : https://prnt.sc/K2CP3hLYPwtZ
5. alignment and Custom positioning : https://prnt.sc/BHfWyw5S00tN
6. Box shadow

<hr>

### Navigation

1. Ability to control color in disabled state : https://prnt.sc/vhoquSR0NDqU (When the carousel has reached the end)
2. arrows box width and height : https://prnt.sc/HTYhyYh46OHC
3. Custom positioning : https://prnt.sc/ZIyrBfADVij_
4. arrows box Background color : https://prnt.sc/NkBE0EALUNSe
5. border and border randios
6. Box shadow

As another useful feature, it would be great if you added the ability to customize Specific Next and Prev Arrow Styles. This feature includes three controls. **Border** and **border radius** + **box shadow**.
Using this option, we can control Next and Prev Arrow style separately. Please pay attention to the image below. As you can see in the image below, the amount of border radius in the two arrow boxes is controlled separately. This is one of the best features in the jet woobuilder plugin carousel.

![1](https://github.com/elementor/elementor/assets/88034902/e87d4421-18ff-4f87-b319-8881b7382fa5)

In general, jet woobuilder plugin > product grid widget is the best example for Pagination and Navigation development in Elementor. Please keep these things in mind.

**Comment by Joliamendi** [comment]
+1
Let me add one thing to Pagination.

- Pagination can be displayed vertically


![one](https://github.com/elementor/elementor/assets/103376243/03b3c486-2144-41f9-9b16-94b0b51af824)

**Comment by hein-obox** [comment]
I love this!

**Reply by Michaelgimii** [reply]
Hi @hein-obox, Nice to see you here. I hope you work on them💪

**Comment by nicholaszein** [comment]
Hi @Michaelgimii!

thanks for submitting your feature request! 🙏 

### 🗃️ Please understand that while we do welcome new feature requests, we can't track multiple requests per thread. 

Please follow our guidelines for submitting a new [🚀 Feature Request](https://github.com/elementor/elementor/discussions/new?category=feature-request), and submit only ☝️ **_<ins>one request per thread</ins>_** so we can track them better and allow users to vote only for the features they want to see implemented.

Kind regards
